src/datasets/load.jl

```diff
@@ -1,42 +1,42 @@
 
 """
-    datasetpath(name)
+datasetpath(name)
 
 Return the folder that dataset `name` is stored.
 
 If it hasn't been downloaded yet, you will be asked if you want to
 download it. See [`Datasets.DATASETS`](#) for a list of available datasets.
 """
 function datasetpath(name)
```

Comment:
Looks like the indentation got a little messed up in this file?

Reply:
Just to clarify, is there a way to auto format the code ?